Application/Transfer/Import/MLS/Service.php

```diff
@@ -0,0 +1,945 @@
+<?php
+
+namespace SPHERE\Application\Transfer\Import\MLS;
+
+use MOC\V\Component\Document\Component\Bridge\Repository\PhpExcel;
+use MOC\V\Component\Document\Document;
+use SPHERE\Application\Contact\Address\Address;
+use SPHERE\Application\Education\Lesson\Division\Division;
+use SPHERE\Application\Education\School\Type\Type;
+use SPHERE\Application\People\Group\Group;
+use SPHERE\Application\People\Meta\Common\Common;
+use SPHERE\Application\People\Meta\Common\Service\Entity\TblCommonBirthDates;
+use SPHERE\Application\People\Meta\Common\Service\Entity\TblCommonInformation;
+use SPHERE\Application\People\Meta\Prospect\Prospect;
+use SPHERE\Application\People\Person\Person;
+use SPHERE\Application\People\Relationship\Relationship;
+use SPHERE\Application\Setting\Consumer\Consumer;
+use SPHERE\Common\Frontend\Form\IFormInterface;
+use SPHERE\Common\Frontend\Layout\Repository\Panel;
+use SPHERE\Common\Frontend\Layout\Structure\Layout;
+use SPHERE\Common\Frontend\Layout\Structure\LayoutColumn;
+use SPHERE\Common\Frontend\Layout\Structure\LayoutGroup;
+use SPHERE\Common\Frontend\Layout\Structure\LayoutRow;
+use SPHERE\Common\Frontend\Message\Repository\Danger;
+use SPHERE\Common\Frontend\Message\Repository\Success;
+use SPHERE\Common\Frontend\Message\Repository\Warning;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use SPHERE\Application\Transfer\Import\Service as ImportService;
+
+/**
+ * Class Service
+ *
+ * @package SPHERE\Application\Transfer\Import\MLS
+ */
+class Service
+{
+    /**
+     * @param IFormInterface|null $Form
+     * @param UploadedFile|null $File
+     *
+     * @return IFormInterface|Danger|string
+     */
+    public function createStudentsFromFile(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method createStudentsFromFile() has 393 lines of code. Current threshold is set to 100. Avoid really long methods.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)